src/formats/MMTF.cpp

```diff
@@ -24,6 +24,7 @@
 #include "chemfiles/Connectivity.hpp"
 
 #include "chemfiles/types.hpp"
+#include "chemfiles/parse.hpp"
```

Comment:
Is this needed? Maybe for `trim`, but that should be handled by the external library.